prov/verbs/src/ep_rdm/verbs_rdm_cm.c

```diff
@@ -615,21 +617,76 @@ ssize_t fi_ibv_rdm_conn_cleanup(struct fi_ibv_rdm_conn *conn)
 	return ret;
 }
 
+static int fi_ibv_rdm_poll_cq(struct fi_ibv_rdm_ep *ep)
```

Comment:
Thanks, we'll remove this prefix to avoid name collision.

Seems, I got it correctly, I meant the same. Yes, `fi_ibv_rdm_tagged_poll `really skips CQ polling if ep was closed. But it doesn't look as really so what I need, since ep_closing flags is set when EP is closed. In my case I need to check `fi_ibv_rdm_conn::state`.